charts/osdfir-infrastructure/README.md

```diff
@@ -0,0 +1,295 @@
+<!--- app-name: OSDFIR Infrastructure -->
+# OSDFIR Infrastructure Helm Chart
+
+OSDFIR Infrastructure helps setup Open Source
+Digital Forensics tools to Kubernetes clusters using Helm. 
+
+Currently, OSDFIR Infrastructure
+supports the deployment of the following tools:
+  * Timesketch; ref https://github.com/google/timesketch
+  * Turbinia; ref https://github.com/google/turbinia
+
+## TL;DR
+
+```console
+helm install osdfir-release oci://us-docker.pkg.dev/osdfir-registry/osdfir-charts/osdfir-infrastructure
+```
+> **Tip**: To quickly get started with a local cluster, see [minikube install docs](https://minikube.sigs.k8s.io/docs/start/).
+
+## Introduction
+
+This chart bootstraps a OSDFIR Infrastructure deployment on a [Kubernetes](https://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- Kubernetes 1.19+
+- Helm 3.2.0+
+- PV provisioner support in the underlying infrastructure
+
+> **Note**: Currently Turbinia only supports processing of GCP Persistent Disks and Local Evidence. See [GKE Installations](#gke-installations) for deploying to GKE.
```

Comment:
I'm just trying to note that Turbinia can run on other environments but it won't be able to process cloud based disks except for GCP and any other evidence in case someone for example tries to put it on AWS  and it runs but I guess at that point they will find that out when they try and process a AWS disk there is no option in the CLI to. Should I just point to Turbinia doc for compatibility info?

Reply:
Yeah, pointing to Turbinia docs sounds good, thanks.